src/typescript/frontend/src/components/pages/home/components/table-card/TableCard.tsx

```diff
@@ -129,7 +127,14 @@ const TableCard = ({
   // By default set this to 0, unless it's currently the left-most border. Sometimes we need to show a temporary border
   // though, which we handle in the layout animation begin/complete callbacks and in the outermost div's style prop.
   // Always show the left border when there's something in the search bar.
-  const borderLeftWidth = useMotionValue(curr.col === 0 ? 1 : anySearchBytes ? 1 : 0);
+  const borderLeftWidth = useMotionValue(curr.col === 0 ? 1 : 0);
+
+  useEffect(() => {
+    if (curr.col === 0) {
+      borderLeftWidth.set(1);
+    }
+    /* eslint-disable react-hooks/exhaustive-deps */
+  }, [curr]);
```

Comment:
I have a sneaking suspicion that this was only necessary because you changed the `router.refresh()` in the other file, can you make sure that that's not the case?

Reply:
![image](https://github.com/user-attachments/assets/5222b2c0-bea0-45d0-b503-ad13c344edb2)

This is a screenshot from main, where that change is not present. See https://emojicoin-dot-fun-git-main-econia-labs.vercel.app/?q=0xf09f9095

Reply:
Awesome thanks for checking this- I thought I fixed this a while ago but I guess not, thanks for checking

Also what is that blue checkmark on the picture?